components/Checkbox/src/index.scss

```diff
@@ -1,118 +1,10 @@
-}
-
-.denhaag-checkbox__input:checked + .denhaag-checkbox__icon {
-  background-color: var(--utrecht-checkbox-checked-background-color);
-  border-width: var(--utrecht-checkbox-checked-border-width);
-  border-color: var(--utrecht-checkbox-checked-border-color);
-}
-
-.denhaag-checkbox__input:checked + .denhaag-checkbox__icon .denhaag-icon {
-  display: block;
-  font-size: var(--utrecht-checkbox-icon-size);
-  color: var(--utrecht-checkbox-checked-color);
-}
-
-.denhaag-checkbox.denhaag-checkbox--error .denhaag-checkbox__icon {
-  border-width: var(--utrecht-checkbox-invalid-border-width, var(--utrecht-form-control-invalid-border-width,));
-  border-color: var(--utrecht-checkbox-invalid-border-color, var(--utrecht-form-control-invalid-border-color));
-}
+@import "~@utrecht/components/checkbox/css";
 
-.denhaag-checkbox.denhaag-checkbox--error .denhaag-checkbox__icon .denhaag-icon {
-  color: var(--utrecht-checkbox-invalid-color);
-}
-
-.denhaag-checkbox:focus-within .denhaag-checkbox__icon,
-.denhaag-checkbox.denhaag-checkbox--focus .denhaag-checkbox__icon {
-  outline: var(--denhaag-focus-border);
-}
-
-.denhaag-checkbox:hover .denhaag-checkbox__icon,
-.denhaag-checkbox.denhaag-checkbox--hover .denhaag-checkbox__icon,
-.denhaag-form-control-label:hover .denhaag-checkbox__icon {
-  border-width: var(--denhaag-checkbox-hover-border-width);
-  border-color: var(--denhaag-checkbox-hover-border-color);
-}
-
-.denhaag-checkbox .denhaag-checkbox__input:disabled + .denhaag-checkbox__icon {
-  background-color: var(--utrecht-checkbox-disabled-background-color);
-  border-width: var(--utrecht-checkbox-disabled-border-width);
-  border-color: var(--utrecht-checkbox-disabled-border-color);
-  cursor: default;
-}
-
-.denhaag-checkbox.denhaag-checkbox--error:hover .denhaag-checkbox__icon,
-.denhaag-checkbox.denhaag-checkbox--error.denhaag-checkbox.denhaag-checkbox--hover .denhaag-checkbox__icon,
-.denhaag-form-control-label:hover .denhaag-checkbox.denhaag-checkbox--error .denhaag-checkbox__icon {
-  border-color: var(--denhaag-checkbox-hover-invalid-border-color);
-}
-
-.denhaag-checkbox:hover .denhaag-checkbox__input:checked + .denhaag-checkbox__icon,
-.denhaag-checkbox.denhaag-checkbox--hover .denhaag-checkbox__input:checked + .denhaag-checkbox__icon,
-.denhaag-form-control-label:hover .denhaag-checkbox__input:checked + .denhaag-checkbox__icon {
-  background-color: var(--denhaag-checkbox-hover-checked-background-color);
-  border-color: var(--denhaag-checkbox-hover-checked-border-color);
-}
-
-.denhaag-checkbox .denhaag-checkbox__input:checked:disabled + .denhaag-checkbox__icon {
-  background-color: var(--denhaag-checkbox-checked-disabled-background-color);
-  border-color: var(--denhaag-checkbox-checked-disabled-border-color);
-}
-
-.denhaag-checkbox.denhaag-checkbox--error .denhaag-checkbox__input:checked + .denhaag-checkbox__icon {
-  background-color: var(--denhaag-checkbox-checked-invalid-background-color);
-}
+.utrecht-checkbox--custom {
```

Comment:
Zou dit `--denhaag` kunnen zijn?

Reply:
@Robbert Ik begrijp niet precies wat je bedoelt.

Heb nog even gekeken, en volgens mij zit het probleem in het component bij Utrecht, er is wel een --utrecht-checkbox-border-radius design token, maar die werkt niet. Ik kan wel even bij Utrecht een issue of PR aanmaken, dat probeer ik morgen of volgende week wel even te doen.